docs/developers/relayed-transactions.md

```diff
@@ -241,3 +244,130 @@ Decoding the arguments ([useful resources here](/developers/sc-calls-format/)) w
 - 2nd argument: `0f` => `15` (nonce of the inner transaction)
 - 3rd argument: `616464` => `add` (function to be called - no argument needed in this example)
 - 4th argument: `9abd13f4f53f3f23597744d65fc5d351bb6773e05a954b419c08d18593d1c6926279a4d16144d3f866a5487803142fcff0eab6ad588209969acb7abed210200b` (the signature of the inner transaction)
+
+[comment]: # (mx-context-auto)
+
+## Relayed transactions version 3
+
+:::note
+This feature is not available at the moment on Mainnet.
+:::
+
+Relayed transactions v3 feature comes with a change on the entire transaction structure, adding a new optional field `InnerTransactions`, which is a slice of pointers to the Transaction structure. That being said, relayed transactions v3 allow the user to send multiple inner transactions on the same relayed transaction which will be executed as normal transactions, without the gas consuming data field of the old relayed transactions versions.
```

Comment:
> which is a slice of pointers to the Transaction structure

_... which is a collection of (inner, nested) transactions ..._

(generally speaking, people do not know what is a pointer, what is an array slice. For the more technical, you can use the phrase "array of transactions", but not "slice").